cllc-public-app/Models.Extensions/Contact.cs

```diff
@@ -161,8 +165,10 @@ public static void CopyHeaderValues(this ViewModels.Worker to, IHeaderDictionary
 
             // birthdate is YYYY-MM-DD
             string smgov_birthdate = headers["smgov_birthdate"];
+            smgov_birthdate = "1989-02-21";
             // Male / Female / Unknown. 
             string smgov_sex = headers["smgov_sex"];
+            smgov_sex = "Female";
```

Comment:
This looks like it should be removed before we merge the PR

Reply:
good catch thanks

Reply:
fixed